base/data/gm4/tags/blocks/waterloggable.json

```diff
@@ -1,25 +1,51 @@
 {
   "values" : [
+    "#minecraft:campfires",
+    "#minecraft:fences",
     "#minecraft:signs",
     "#minecraft:slabs",
     "#minecraft:stairs",
     "#minecraft:trapdoors",
     "#minecraft:underwater_bonemeals",
```

Comment:
#minecraft:underwater_bonemeals should not be included, as this tag includes seagrass, which can't be waterlogged. This should be replaced with #minecraft:corals and #minecraft:wall_corals